resources/views/index.blade.php

```diff
@@ -40,28 +45,56 @@
-                <div v-for="star in [5,4,3,2,1]">
-                    <input v-model="reviewParams.stars" type="radio" :value="star">
-                    <v-star :value="star"></v-star>
+                <div class="stars">
+                <span class="star font">
+                    <span>
+                        {{$detail->review > 1 ? '★' : '☆' }}
+                    </span>
+                    <span>
+                        {{$detail->review  > 2 ? '★' : '☆' }}
+                    </span>
+                    <span>
+                        {{$detail->review  > 3 ? '★' : '☆' }}
+                    </span>
+                    <span>
+                        {{$detail->review  > 4 ? '★' : '☆' }}
+                    </span>
+                    <span>
+                        {{$detail->review  > 5 ? '★' : '☆' }}
+                    </span>
+                </span>
+                <span class="value font">{{$detail->review}}</span>
                 </div>
-                {{-- ↑ここの情報の引っ張り方は村田さんに確認する --}}
                 <div class="button">
-                <button><a href="{{ route('stores.show', $detail->id)}}">詳細（どう飛ぶ？？）</a></button>
+                <button><a href="{{route('stores.show', $detail->id}})">詳細</a></button>
                 </div>
             </div> 
             <div class="usercoment font">
                 <div class="userinfo">
                 <p>{{ $detail->bestreview()['name'] }}さんの口コミ</p>
                 <p class="add_time">投稿日時：{{ $detail->bestreview()['created_at'] }}</p>
                 </div>
-                <div v-for="star in [5,4,3,2,1]">
-                    <input v-model="reviewParams.stars" type="radio" :value="star">
-                    <v-star :value="star"></v-star>
-                </div>
-                <p>{{ $detail->bestreview()['review'] }}</p>
+                <span class="star font">
+                    <span>
+                        {{$detail->bestreview()['review'] > 1 ? '★' : '☆' }}
+                    </span>
+                    <span>
+                        {{$detail->bestreview()['review'] > 2 ? '★' : '☆' }}
+                    </span>
+                    <span>
+                        {{$detail->bestreview()['review'] > 3 ? '★' : '☆' }}
+                    </span>
+                    <span>
+                        {{$detail->bestreview()['review'] > 4 ? '★' : '☆' }}
+                    </span>
+                    <span>
+                        {{$detail->bestreview()['review'] > 5 ? '★' : '☆' }}
+                    </span>
+                </span>
+                <span class="value font">{{$detail->bestreview()['review']}}</span>
                 <p>{{ $detail->bestreview()['comment'] }}</p>
                 <div class="button">
-                <button><a href="{{ route('reviews.show', $detail->bestreview()['id'])}}">続きを見る・・・（どう飛ぶ？）</a></button>
+                <button><a href="{{route('stores.show', $detail->bestreview(['id']))}}">続きを見る</a></button>
```

Comment:
[MUST]
お店の詳細ページに行くので、引数で渡すIDはお店のIDです！

```suggestion
                <button><a href="{{ route('stores.show', $detail->id) }}">続きを見る</a></button>
```